rollup/rollup.config.js

```diff
@@ -0,0 +1,24 @@
+import resolve from '@rollup/plugin-node-resolve';
+import commonjs from '@rollup/plugin-commonjs';
+import typescript from '@rollup/plugin-typescript';
+import json from '@rollup/plugin-json';
+import builtins from 'builtin-modules';
+
+export default {
+  input: 'src/index.ts',
+  output: {
+    file: 'dist/index.js',
+    format: 'cjs',
+  },
+  plugins: [
+    resolve({ preferBuiltins: true }),
+    commonjs(),
+    json(),
```

Comment:
```suggestion
    json({ compact: true }),
```
So the ABIs don't take up so much room in the output